docs/topics/conditional-formatting.md

```diff
@@ -653,11 +653,11 @@ $wizardFactory = new Wizard($cellRange);
 /** @var Wizard\Expression $expressionWizard */
 $expressionWizard = $wizardFactory->newRule(Wizard::EXPRESSION);
 
-$expressionWizard->expression('ISODD(A1)')
+$expressionWizard->expression('ISODD(A2)')
```

Comment:
In Conditional Formulae, we always use cell A1 to reference the top-left cell in the formula range, whatever that range might be.
> Because we use cell `A1` as the baseline cell for relative references, the Wizard is able to handle the necessary adjustments for cell references and formulae to match the range of cells that it is being applied to when `getConditional()` is called, so it returns the correct expression.

So I'm afraid this is actually changing the formula to reference a different cell... basically saying if the cell in the row below is even or odd